dev/com.ibm.ws.jaxrs.2.1_fat_extended/test-applications/subResourceApp/src/jaxrs21/fat/subresource/MyBean.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * 
+ */
```

Comment:
nitpicky but I would prefer to see empty comment blocks deleted

Reply:
good catch - this was supposed to be the copyright header - I'll change it.